scripts/make_credentials.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+C4GH_VERSION="1.7.3"
+
+apt-get -o DPkg::Lock::Timeout=60 update > /dev/null
+apt-get -o DPkg::Lock::Timeout=60 install -y curl postgresql-client openssl > /dev/null
+
+for n in download finalize inbox ingest mapper sync verify; do
+    echo "creating credentials for: $n"
+    ## password and permissions for MQ
+    curl -s -u test:test -X PUT "http://rabbitmq:15672/api/users/$n" -H "content-type:application/json" -d '{"password": "'"$n"'", "tags":"none"}'
```

Comment:
This will fail if the password contains quoting characters.  Better use `jq` to create the JSON.

```shell
body_data=$( jq -n -c --arg password "$n" --arg tags none '$ARGS.named' )
```
... then use that with `curl -d`.